py3-msgspec.yaml

```
@@ -0,0 +1,61 @@
+package:
+  name: py3-msgspec
+  version: "0.19.0"
+  epoch: 0
+  description: Fast serialization and validation library for JSON, MessagePack, YAML, and TOML
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    provider-priority: 0
+
+vars:
+  pypi-package: msgspec
+
+data:
+  - name: py-versions
+    items:
+      3.10: '310'
+      3.11: '311'
+      3.12: '312'
+      3.13: '313'
+
+environment:
+  contents:
+    packages:
+      - py3-supported-build-base-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/jcrist/msgspec.git
+      tag: ${{package.version}}
+      expected-commit: dd965dce22e5278d4935bea923441ecde31b5325
+
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package}} installed for python${{range.key}}
+    pipeline:
+      - uses: py/pip-build-install
+        with:
+          python: python${{range.key}}
+      - uses: strip
+    test:
+      pipeline:
+        - uses: python/import
+          with:
+            python: python${{range.key}}
+            import: ${{vars.pypi-package}}
+
```

We can add a minimal functional test for the library (Not a blocker):
```suggestion

        - name: Test some simple date/time functions
          runs: |
            cat >> test.py <<EOF
            import ${{vars.pypi-package}}

            class User(msgspec.Struct):
                id: int
                name: str
                email: str
                active: bool = True
                
            user = User(id=1, name="test", email="test@example.com")
            encoded = msgspec.json.encode(user)
            print("Encoded JSON:", encoded)
            EOF

            python${{range.key}} test.py | grep -q '"id":1,"name":"test","email":"test@example.com","active":true'
```